src/main/app/form/validation.ts

```diff
@@ -172,7 +172,7 @@ export const isPhoneNoValid: Validator = value => {
 
 export const isAlphaNumeric: Validator = value => {
```

Comment:
@sealDebolina We can't change this common validator, it will affect all places.
Better to use any existing validators with ' or add a new one please.

Reply:
@channa-patil in new build added new validation "isAlphaNumericWithApostrophe" and reverted back "isAlphaNumeric".
But have not changed any content.ts file where Apostrophe is required(asked by Elize), as if security doesnot allow apostrophe then we may require to revart them back again